docs/build/reading.md

```diff
@@ -4,20 +4,22 @@ Use the [`idx.get()`](../reference/idx.md#get) method to query a [record](../lea
 
 ## **Using default aliases**
 
-Pass an alias from [default definitions](../guides/definitions/default.md) and the DID that you wish to query.
+Pass an alias from [default definitions](../guides/definitions/default.md), and the [DID](../learn/glossary.md#did) or [Blockchain Account ID](../learn/glossary.md##caip-10-account-id) that you wish to query.
```

Comment:
Looks like there's a double `#` here in `glossary.md##caip-10-account-id`, does the link work as expected?

Reply:
woops